draft-barnes-hpke-hpke.md

```diff
@@ -436,6 +453,15 @@ for the Diffie-Hellman group in use. {{derive-key-pair}} contains the
 `DeriveKeyPair()` function specification for DHKEMs defined in this document.
 
 ~~~
+# For use with single-stage KDFs
+def ExtractAndExpand(dh, kem_context):
+  derive_input = concat(
+    lengthPrefixed(dh),
+    lengthPrefixed(shared_secret)
```

Comment:
What is `shared_secret`? It does not seem to be (implicit) parameter to `ExtractAndExpand()`, and looking at the code, using the function, looks like shared_secret is the output of it.

Reply:
I think it's a typo for `kem_context`.